tests/coords/transform_coords_test.py

```diff
@@ -951,3 +951,55 @@ def test_prioritize_coords_attrs_conflict(a):
 
     with pytest.raises(sc.DataArrayError):
         original.transform_coords(['b'], graph={'b': 'a'})
+
+
+def test_keyword_syntax_equivalent_to_explicit_syntax():
+    da = sc.data.table_xyz(nrow=10)
+
+    def a(x):
+        return x + x
+
+    def b(x, y):
+        return x + y
+
+    assert sc.identical(da.transform_coords(a=a), da.transform_coords('a', {'a': a}))
+    graph = {'a': a, 'b': b}
+    assert sc.identical(da.transform_coords(a=a, b=b),
+                        da.transform_coords(['a', 'b'], graph=graph))
+
+
+def test_keyword_syntax_without_entries_returns_unchanged():
+    da = sc.data.table_xyz(nrow=10)
+    assert sc.identical(da.transform_coords(), da)
+
+
+def test_keyword_syntax_without_entries_and_graph_returns_unchanged():
+    da = sc.data.table_xyz(nrow=10)
+    assert sc.identical(da.transform_coords(graph={'b': 'y'}), da)
```

Comment:
My reasoning was:
- No harm done?
- No extra code required.
- May allow for generic code.

Reply:
I find it a bit strange because we are distinguishing between `targets=None` and `targets=[]` to determine if keyword arguments are allowed. But here, the two are equivalent.
It is not a big issue and I am fine with leaving it as is.